tests/unittest/executor/test_sleep_collective_rpc_guards.py

```
@@ -93,28 +93,407 @@ def test_missing_sleep_config_raises(self, method):
+
+        # Two peers return ok; we only care that recv() is called for both.
+        responses = [{"status": "ok"}, {"status": "ok"}]
+        w, recv_calls = _make_proto_worker(responses, world_size=3)
+
+        with patch(
+            "tensorrt_llm._torch.virtual_memory.release_with_tag",
+            side_effect=RuntimeError("rank 0 VMM fault"),
+        ), patch(
+            "tensorrt_llm._torch.virtual_memory.materialize_with_tag"
+        ), patch("torch.cuda.synchronize"), \
+             patch("gc.collect"), \
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)